marsha/core/tests/test_api_video.py

```diff
@@ -65,7 +65,7 @@ def test_api_video_read_detail_token_user(self):
                 "description": video.description,
                 "id": str(video.id),
                 "title": video.title,
-                "urls": {"jpeg": jpeg_dict, "mp4": mp4_dict},
+                "urls": json.dumps({"mp4": mp4_dict, "jpeg": jpeg_dict}),
```

Comment:
It has changed to (I forgot to change it in the serializer but maybe you can already take it into account and rebase on the PR I'm preparing now so we don't have to change it just after adding it?) :
```javascript
{"mp4": mp4_dict, "thumbnails": thumbnails_dict, "previews": previews_dict}
```

Reply:
@mbenadda no more previews...